zerver/webhooks/github/tests.py

```diff
@@ -479,6 +479,18 @@ def test_ignored_pull_request_actions(self) -> None:
             payload = orjson.dumps(data).decode()
             self.verify_post_is_ignored(payload, "pull_request")
 
+    def test_ignored_pull_request_review_actions(self) -> None:
+        ignored_actions = [
+            "edited",
+        ]
+        for action in ignored_actions:
+            # empty changes to simulate a submitted pr review that should not send a message
+            data = dict(action=action, changes=dict())
+            payload = orjson.dumps(data).decode()
+            self.verify_post_is_ignored(payload, "pull_request_review")
+
+        self.check_webhook("pull_request_review_duplicate_payload", "", "", expect_noop=True)
```

Comment:
> **sbansal1999** : We can probably add a fixture for this test. Feel free to use the one that was discussed [here](https://chat.zulip.org/#narrow/stream/127-integrations/topic/github.20bot/near/1600140) or you can generate a new one as well.

additionally, I've extended the `test_ignored_pull_request_review_actions` test to use the `check_webhook` function with the duplicate payload. Do we need to keep the test logic using `verify_post_is_ignored` here? it feels redundant but also consistent with all other ignored related tests.

Reply:
Since we are not completely ignoring all the `edited` events, I feel that `ignored_actions` array gives a wrong indication. Something like this would be a better solution, in my opinion:

```python
   def test_pull_request_review_edited_with_empty_changes_ignore(self) -> None:
        payload = self.get_body("pull_request_review__edited_empty_changes")
        self.verify_post_is_ignored(payload, "pull_request_review")
```

Reply:
Looks like one comment got deleted smh,

Since we are not completely ignoring all the `edited` events, I feel that `ignored_actions` array gives a wrong indication. Something like [this](https://github.com/zulip/zulip/blob/main/zerver/webhooks/github/tests.py#L465-L467) would be a better solution, in my opinion.

Reply:
Ah, okay, that makes more sense.